tmt/base.py

```diff
@@ -496,6 +497,21 @@ def __init__(self, node, run=None):
             (key, str(value)) for key, value
             in node.get('environment', dict()).items()])
 
+        # Environment variables from files
+        env_files = self.node.get("environment-file") or []
+        assert isinstance(env_files, list), f"environment_file parameter should be a list. " \
```

Comment:
We should return nicer error to the user instead of traceback. I can't find better exception then `tmt.utils.GeneralError`.

This way tmt exists with non-zero and prints error message (string from the exception).

Reply:
okay